### PR TITLE
test(accordion): add interaction tests for multi-active mode

### PR DESCRIPTION
**Pull request by niksa-monday**
Add interaction tests for multi-active mode
 - add interactions tests
 - connect tests to storybook

Minor refactoring 
 - improve interaction tests of single active
 - split to helper/check functions

### CODE REVIEW

File: src/components/Accordion/Accordion/__tests__/accordion.interactions.js [hunk 1]
```diff
@@ -14,41 +14,89 @@ function getOpenedAccordionItem(canvas) {
...
+  const before = getOpenedAccordionItem(canvas);
+  const elHeading = getAccordionHeadingByText(canvas, title);
+  userEvent.click(elHeading);
+  const elPanel = canvas.getByRole("region");
+  await expect(elHeading.getAttribute("aria-expanded")).toBe("true");
+  await expect(before.elHeading.getAttribute("aria-expanded")).toBe("false");
+  await expect(elHeading.getAttribute("aria-controls")).toBe(elPanel.id);
+}
+
+async function closeAndCheckMultiAccordionItem(canvas, shouldBeOpened) {
```
Comment (laviomri) [inline src/components/Accordion/Accordion/__tests__/accordion.interactions.js:35]: the name `shouldBeOpened` confused me - I thought it was a boolean... how about `expectedOpenedPanels` or something?

File: src/components/Accordion/Accordion/__tests__/accordion.interactions.js [hunk 2]
```diff
@@ -14,41 +14,89 @@ function getOpenedAccordionItem(canvas) {
...
+  const before = getOpenedAccordionItems(canvas);
+  userEvent.click(before.elHeadings[0]);
+  const after = getOpenedAccordionItems(canvas);
+  await expect(after.elPanels.length).toBe(shouldBeOpened);
+  await expect(before.elHeadings[0].getAttribute("aria-expanded")).toBe("false");
+}
+
+async function openAndCheckMultiAccordionItem(canvas, title, shouldBeOpened) {
```
Comment (laviomri) [inline src/components/Accordion/Accordion/__tests__/accordion.interactions.js:43]: same comment about param name

File: src/components/Accordion/Accordion/__tests__/accordion.interactions.js [hunk 3]
```diff
@@ -14,41 +14,89 @@ function getOpenedAccordionItem(canvas) {
...
-  elPanel = canvas.getByRole("region");
-  await expect(elHeading.getAttribute("aria-expanded")).toBe("true");
-  await expect(before.elHeading.getAttribute("aria-expanded")).toBe("false");
-  await expect(elHeading.getAttribute("aria-controls")).toBe(elPanel.id);
+  await delay(CHANGES_DELAY);
+  await openAndCheckAccordionItem(canvas, "Setting");
+};
+
+const closeAlreadyActiveMultiActiveTests = async canvas => {
+  await delay(CHANGES_DELAY);
```
Comment (laviomri) [inline src/components/Accordion/Accordion/__tests__/accordion.interactions.js:74]: great, now it feels much more clear what the test is doing 👍 👍

**Review by laviomri — APPROVED** [review]

File: src/components/Accordion/Accordion/__tests__/accordion.interactions.js (hunk 1 above)
Comment (niksa-monday) [reply src/components/Accordion/Accordion/__tests__/accordion.interactions.js]: Changed 👍

File: src/components/Accordion/Accordion/__tests__/accordion.interactions.js (hunk 2 above)
Comment (niksa-monday) [reply src/components/Accordion/Accordion/__tests__/accordion.interactions.js]: Changed

**Pull request merged**